examples/example_tasklet.c

```diff
@@ -2,7 +2,6 @@
  * example_tasklet.c
  */
 #include <linux/delay.h>
-#include <linux/interrupt.h>
```

Comment:
Do not remove this header since the example still uses the functions/macros, e.g., `tasklet_kill()`, in this header.

Reply:
And, I think this is why GitHub Actions is yelling.

Reply:
Indeed. That's why as I said the examples needs to be revisited :-)
Thank you for your reviews!